src/components/itemCard.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { useToggle } from 'react-hooks-lib'
+
+export default function ItemCard({ item }) {
+  const [isDetailsVisible, setIsDetailsVisible] = useState(false)
```

Comment:
I guess this state isDetailsVisible repeats the useToggle functionality. Leave it only if you want to use it later. Even then, the useToggle still repeats the functionality of having to modi (on and off) and you still can use this state.